### PR TITLE
[sushy_emulator] Make instance whitelist optional

### PR DESCRIPTION
**Pull request by lewisdenny**

### CODE REVIEW

**Comment by lewisdenny** [comment]
recheck

**Comment by lewisdenny** [comment]
recheck

**Review by marios — APPROVED** [review]
came via slack request, would be nice to have some more info in the commit message but guessing based on the commit title, indeed it moves tasks into a block and executes only when uuid file exists

File: roles/sushy_emulator/tasks/collect_details.yml [hunk 1]
```diff
@@ -69,20 +69,30 @@
...
     - name: "Slurp content of: {{ _uuid_file }}"
-      ansible.builtin.slurp:
-        src: "{{ _uuid_file }}"
-      register: _libvirt_uuids_file
+      when: _uuid_file_status.stat.exists
+      block:
+        - name: "Slurp content of: {{ _uuid_file }}"
+          ansible.builtin.slurp:
+            src: "{{ _uuid_file }}"
+            register: _libvirt_uuids_file
```
Comment (pablintino) [inline roles/sushy_emulator/tasks/collect_details.yml:83]: This register is wrongly indented.

Comment (lewisdenny) [reply roles/sushy_emulator/tasks/collect_details.yml:83]: Interesting the linter didn't catch this :(

Comment (lewisdenny) [reply roles/sushy_emulator/tasks/collect_details.yml:83]: Done

**Comment by pablintino** [comment]
@frenzyfriday care to add your approval after the small tweak Lewis did?

File: roles/sushy_emulator/tasks/collect_details.yml (hunk 1 above)
Comment (frenzyfriday) [reply roles/sushy_emulator/tasks/collect_details.yml:83]: haha, I missed that as well 😅

**Pull request merged**